src/overlays/actors/ovl_En_Viewer/z_en_viewer.c

```diff
@@ -19,29 +19,32 @@
 
 #define THIS ((EnViewer*)thisx)
 
+#define TYPE (this->actor.params >> 8)
+
```

Comment:
we still havent standardized how we are going to do this, so it would probably be best to wait until we do so. At the very least it would be a macro like GET_TYPE that takes params as an arg. but theres still some details to figure out with that